### PR TITLE
Re-enable Kafka ssl sasl tests

### PR DESCRIPTION
**Pull request by zhfeng**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java [hunk 1]
```diff
@@ -16,10 +16,10 @@
  */
 package org.apache.camel.quarkus.kafka.sasl;
 
-import io.quarkus.test.junit.DisabledOnNativeImage;
 import io.quarkus.test.junit.NativeImageTest;
+import org.junit.jupiter.api.Disabled;
 
-@DisabledOnNativeImage("https://github.com/apache/camel-quarkus/issues/3239")
+@Disabled("https://github.com/apache/camel-quarkus/issues/2729")
```
Comment (jamesnetherton) [inline integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java:22]: Need a new issue, that one was closed and fixed some time ago.

Comment (zhfeng) [reply integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java:22]: yeah, I will raise a new issue to fix these native tests for ```KafkaSaslSslIT``` and ```KafkaSslTest```.

Comment (zhfeng) [reply integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java:22]: and it should be fixed in main branch ?

Comment (jamesnetherton) [reply integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java:22]: Ah you're right. It was already disabled on the `main` branch so I guess we should fix it there.

In that case we can probably merge this PR and take care of `KafkaSaslSslIT` on the `main` branch.

Comment (zhfeng) [reply integration-tests/kafka-sasl-ssl/src/test/java/org/apache/camel/quarkus/kafka/sasl/KafkaSaslSslIT.java:22]: yeah, I raise https://github.com/apache/camel-quarkus/issues/3254

**Pull request merged**